witnet.toml

```diff
@@ -2,7 +2,36 @@
 # https://docs.witnet.io/configuration/toml-file/
 [connections]
 server_addr = "0.0.0.0:21337"
-known_peers = ["52.166.178.145:21337", "52.166.178.145:22337", "52.166.178.145:23337", "52.166.178.145:24337", "52.166.178.145:25337"]
+public_addr = "0.0.0.0:21337"
```

Comment:
A comment here would be helpful, to explain the difference between `server_addr` and `public_addr`.